### PR TITLE
SERVER-2047 | added ttlSecondsAfterFinished in job

### PR DESCRIPTION
**Pull request by atulsingh0**
:gear: **Issue**

[SERVER-2047](https://circleci.atlassian.net/browse/SERVER-2047)

:white_check_mark: **Fix**

* Added TTL for migrator job
* Increased wait to 10 mins
* Deleted the Job deletion commands

:question: **Tests**

- [ ] Tested Updating Existing Instance
- [ ] Installed on new instance

### CODE REVIEW

File: kots-exporter/templates/circle-migrator.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ metadata:
     app: circle-migrator
     layer: application
 spec:
+  ttlSecondsAfterFinished: 86400   # 24- hrs
```
Comment (jbialy) [inline kots-exporter/templates/circle-migrator.yaml:9]: This is neat. The only interesting thing to note is that the feature become first enabled (beta) by default in Kubernetes 1.21 and is now stable in 1.23. Not sure if we care but for customers on k8s < 1.21 it likely won't work unless they have manually enabled alpha features.

**Review by jbialy — APPROVED** [review]

File: kots-exporter/templates/circle-migrator.yaml (hunk 1 above)
Comment (atulsingh0) [reply kots-exporter/templates/circle-migrator.yaml:9]: In that case, we need to provide the deletion instruction as well.

Comment (atulsingh0) [reply kots-exporter/templates/circle-migrator.yaml:9]: For Server 4, [a minimum k8s 1.22 is required.](https://circleci.com/docs/server/installation/phase-1-prerequisites#create-a-kubernetes-cluster)

**Comment by atulsingh0** [comment]
Re-Open the PR as CCI Server's [minimum req is `k8s v1.24` ](https://circleci.com/docs/server/installation/phase-1-prerequisites#create-a-kubernetes-cluster)

**Review by circlery — APPROVED** [review]
LGTM

**Pull request merged**